<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>scanLimitsTab</class>
 <widget class="QWidget" name="scanLimitsTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>677</width>
    <height>441</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>-20</x>
        <y>0</y>
        <width>663</width>
        <height>441</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QCheckBox" name="filesLargerThanThisCheckBox">
         <property name="text">
          <string>Files larger than this will be skipped and assumed clean</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="filesLargerThanThisSpinBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>70</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>70</height>
          </size>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::PlusMinus</enum>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>999</number>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QComboBox" name="filesLargerThanThisComboBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>70</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>70</width>
           <height>16777215</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string/>
          </property>
         </item>
         <item>
          <property name="text">
           <string>K</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>M</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>G</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="maxAmountForContainerCheckBox">
         <property name="text">
          <string>The maximum amount of data to scan for each container file</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="maxAmountForContainerSpinBox">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::PlusMinus</enum>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>999</number>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QComboBox" name="maxAmountForContainerComboBox">
         <item>
          <property name="text">
           <string/>
          </property>
         </item>
         <item>
          <property name="text">
           <string>K</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>M</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>G</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QCheckBox" name="maxNumberForContainerCheckBox">
         <property name="text">
          <string>The maximum number of files to scan for each container file</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSpinBox" name="maxNumberForContainerSpinBox">
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::PlusMinus</enum>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>999</number>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QComboBox" name="maxNumberForContainerComboBox">
         <item>
          <property name="text">
           <string/>
          </property>
         </item>
         <item>
          <property name="text">
           <string>K</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>M</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>G</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QCheckBox" name="maxArchiveRecursionForContainerCheckBox">
         <property name="text">
          <string>Maximum archive recursion level for container file</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QSpinBox" name="maxArchiveRecursionForContainerSpinBox">
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::PlusMinus</enum>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>999</number>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QComboBox" name="maxArchiveRecursionForContainerComboBox">
         <item>
          <property name="text">
           <string/>
          </property>
         </item>
         <item>
          <property name="text">
           <string>K</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>M</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>G</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QCheckBox" name="maxDirRecursionLevelCheckBox">
         <property name="text">
          <string>Maximum directory recursion level</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QSpinBox" name="maxDirRecursionLevelSpinBox">
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::PlusMinus</enum>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>999</number>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QComboBox" name="maxDirRecursionLevelComboBox">
         <item>
          <property name="text">
           <string/>
          </property>
         </item>
         <item>
          <property name="text">
           <string>K</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>M</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>G</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QCheckBox" name="maxSizeFileForPECheckBox">
         <property name="text">
          <string>Maximum size file to check for embedded PE</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QSpinBox" name="maxSizeFileForPESpinBox">
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::PlusMinus</enum>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>999</number>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="QComboBox" name="maxSizeFileForPEComboBox">
         <item>
          <property name="text">
           <string/>
          </property>
         </item>
         <item>
          <property name="text">
           <string>K</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>M</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>G</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QCheckBox" name="maxSizeHTMLFileToNormalizeCheckBox">
         <property name="text">
          <string>Maximum size of HTML file to normalize</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QSpinBox" name="maxSizeHTMLFileToNormalizeSpinBox">
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::PlusMinus</enum>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>999</number>
         </property>
        </widget>
       </item>
       <item row="6" column="2">
        <widget class="QComboBox" name="maxSizeHTMLFileToNormalizeComboBox">
         <item>
          <property name="text">
           <string/>
          </property>
         </item>
         <item>
          <property name="text">
           <string>K</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>M</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>G</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QCheckBox" name="maxSizeOfNormalizedHTMLFileCheckBox">
         <property name="text">
          <string>Maximum size of normalized HTML file to scan</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QSpinBox" name="maxSizeOfNormalizedHTMLFileSpinBox">
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::PlusMinus</enum>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>999</number>
         </property>
        </widget>
       </item>
       <item row="7" column="2">
        <widget class="QComboBox" name="maxSizeOfNormalizedHTMLFileComboBox">
         <item>
          <property name="text">
           <string/>
          </property>
         </item>
         <item>
          <property name="text">
           <string>K</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>M</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>G</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QCheckBox" name="maxSizeOfScriptFileToNormalizeCheckBox">
         <property name="text">
          <string>Maximum size of script file to normalize</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QSpinBox" name="maxSizeOfScriptFileToNormalizeSpinBox">
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::PlusMinus</enum>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>999</number>
         </property>
        </widget>
       </item>
       <item row="8" column="2">
        <widget class="QComboBox" name="maxSizeOfScriptFileToNormalizeComboBox">
         <item>
          <property name="text">
           <string/>
          </property>
         </item>
         <item>
          <property name="text">
           <string>K</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>M</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>G</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QCheckBox" name="maxSizeZipToTypeReanalzeCheckBox">
         <property name="text">
          <string>Maximum size zip to type reanalyze</string>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="QSpinBox" name="maxSizeZipToTypeReanalzeSpinBox">
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::PlusMinus</enum>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>999</number>
         </property>
        </widget>
       </item>
       <item row="9" column="2">
        <widget class="QComboBox" name="maxSizeZipToTypeReanalzeComboBox">
         <item>
          <property name="text">
           <string/>
          </property>
         </item>
         <item>
          <property name="text">
           <string>K</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>M</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>G</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QCheckBox" name="maxNumberOfPartitionsInDiskImageCheckBox">
         <property name="text">
          <string>Maximum number of partitions in disk image to be scanned</string>
         </property>
        </widget>
       </item>
       <item row="10" column="1">
        <widget class="QSpinBox" name="maxNumberOfPartitionsInDiskImageSpinBox">
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::PlusMinus</enum>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>999</number>
         </property>
        </widget>
       </item>
       <item row="10" column="2">
        <widget class="QComboBox" name="maxNumberOfPartitionsInDiskImageComboBox">
         <item>
          <property name="text">
           <string/>
          </property>
         </item>
         <item>
          <property name="text">
           <string>K</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>M</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>G</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="11" column="0">
        <widget class="QCheckBox" name="maxNumberOfIconsInPEFileCheckBox">
         <property name="text">
          <string>Maximum number of icons in PE file to be scanned</string>
         </property>
        </widget>
       </item>
       <item row="11" column="1">
        <widget class="QSpinBox" name="maxNumberOfIconsInPEFileSpinBox">
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::PlusMinus</enum>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>999</number>
         </property>
        </widget>
       </item>
       <item row="11" column="2">
        <widget class="QComboBox" name="maxNumberOfIconsInPEFileComboBox">
         <item>
          <property name="text">
           <string/>
          </property>
         </item>
         <item>
          <property name="text">
           <string>K</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>M</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>G</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="12" column="0">
        <widget class="QCheckBox" name="numberOfSecondsToWaitForResponseCheckBox">
         <property name="text">
          <string>Number of seconds to wait for waiting a response back from the stats server</string>
         </property>
        </widget>
       </item>
       <item row="12" column="1">
        <widget class="QSpinBox" name="numberOfSecondsToWaitForResponseSpinBox">
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::PlusMinus</enum>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>999</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>writeLimits()</slot>
 </slots>
</ui>
